src/connector.ts

```diff
@@ -662,14 +669,34 @@ async function standaloneConnect(connectSequence: ReplConnectSequence) {
       .analytics()
       .logEvent('REPL', 'StandaloneConnect', `${connectSequence.name} + ${cljsTypeName}`)
       .send();
-    await connect(connectSequence, false).catch(() => {
+    return connect(
+      connectSequence,
+      getConfig().autoSelectNReplPortFromPortFile,
+      hostname,
+      port
+    ).catch(() => {
       // do nothing
```

Comment:
I know this was like this before this PR, but don't we want to do something like log or alert the user if connecting fails?

Reply:
I'll have a look at the call sites. Might be that we should return the promise without this catching.